dbflow-processor/src/main/java/com/raizlabs/android/dbflow/processor/ClassNames.java

```diff
@@ -0,0 +1,117 @@
+package com.raizlabs.android.dbflow.processor;
```

Comment:
@agrosner I am will take care of that now. I am looking at the changed files to resolve other issues that are similar to this one.